firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -1052,7 +1052,7 @@ class TabWebView: WKWebView, MenuHelperInterface, ThemeApplicable {
     }
 
     override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
-        return super.canPerformAction(action, withSender: sender) || action == MenuHelper.SelectorFindInPage
+        return super.canPerformAction(action, withSender: sender) || action == MenuHelperWebViewModel.selectorFindInPage
```

Comment:
Can you clarify why the explicit check for the `selectorFindInPage` is needed?

Reply:
Let me look 👀